web/themes/custom/genlyd_aarhus/genlyd_aarhus.theme

```diff
@@ -225,6 +232,15 @@ function genlyd_aarhus_preprocess_node__activity__teaser(&$variables) {
     $imageUrl = ImageStyle::load('activity_teaser')->buildUrl($image);
   }
 
+  // Set price text.
```

Comment:
This is the same code as above in "genlyd_aarhus_preprocess_node__activity" and it's in the same file, so maybe a helper function ?